includes/definitions/avocent-mp.yaml

```diff
@@ -0,0 +1,11 @@
+os: avocent-mp
+text: Avocent MergePoint Unity
+type: network
+icon: avocent
+discovery:
+    -
+        sysDescr_regex:
```

Comment:
It doesn't look like you need this part? Or is there other devices using same oid?

Reply:
You are right, I should not need the extra sysDescr_regex. I will remove it.